oqum/hazard/00a_source_typologies.tex

```diff
@@ -42,7 +44,18 @@
 
         \item \Gls{charfaultsource} - A typology of source where ruptures
         always fill the entire fault surface.
+        
+        \item \Gls{nonparametricsource} - A typology of source representing a collection of ruptures, each with their associated probabilities of 0, 1, 2 ... occurrences in the investigation time
```

Comment:
For Non Parametric sources I would probably create a separate category since I don't think we can say that ruptures are always covering the whole fault surface for this source type.